apps/server/src/stores/runtimeState.ts

```diff
@@ -79,10 +85,36 @@ export function getState(): Readonly<RuntimeState> {
     runtime: { ...runtimeState.runtime },
     timer: { ...runtimeState.timer },
     _timer: { ...runtimeState._timer },
+    _rundown: { totalDelay: 0 },
   };
 }
 
-export function clear() {
+//TODO: find other things that dose not need clearing
+export function softClear() {
+  runtimeState.eventNow = null;
+  runtimeState.publicEventNow = null;
+  runtimeState.eventNext = null;
+  runtimeState.publicEventNext = null;
+
+  // runtimeState.currentBlock.block = null;
+  // runtimeState.currentBlock.startedAt = null;
```

Comment:
i believe this can bring confusion going forwards, lets comment its value or remove it